src/db.c

```diff
@@ -48,6 +48,7 @@ void updateLFU(robj *val) {
 /* Lookup a key for read or write operations, or return NULL if the key is not
  * found in the specified DB. This function implements the functionality of
  * lookupKeyRead(), lookupKeyWrite() and their ...WithFlags() variants.
+ * The dictEntry reference input is optional, can be used if we already have one.
```

Comment:
didn't we change it to an output arg???

Reply:
yes. updated in https://github.com/redis/redis/pull/13505/commits/77cf67a1d012f45c7db11184eddea50b69d07fe9